device_services/generic/chargepoint.md

```diff
@@ -60,6 +65,22 @@ Chargepoint service is used to represent EV chargers.
 
 > Please note that `ready_to_charge` and `charging` are states required for charging control, while others have only informative value.
 
+* `phase_mode` defines allowed phase balancing modes, see table below for more details:
+
+| Value     | Input EVSE | Output EV  | Grid       |
```

Comment:
Grid here means, that it is only supported in the listed grid types ? Shall be explained...